types/react/index.d.ts

```diff
@@ -777,8 +777,11 @@ declare namespace React {
         defaultProps?: Partial<P> | undefined;
         propTypes?: WeakValidationMap<P> | undefined;
     }
+    
+    /** Alias to fowardRef return.  */
+    type ForwardedComponent = ForwardRefExoticComponent<PropsWithoutRef<P> & RefAttributes<T>>
 
-    function forwardRef<T, P = {}>(render: ForwardRefRenderFunction<T, P>): ForwardRefExoticComponent<PropsWithoutRef<P> & RefAttributes<T>>;
+    function forwardRef<T, P = {}>(render: ForwardRefRenderFunction<T, P>): ForwardedComponent;
```

Comment:
You missed `P` and `T` in `ForwardedComponent`. It should be like `ForwardedComponent<T, P>`